etna/search/views.py

```diff
@@ -526,49 +523,21 @@ def get_selected_filters(self, form: Form) -> Dict[str, List[Tuple[str, str]]]:
         if filter_keyword := form.cleaned_data.get("filter_keyword"):
             return_value.update({"filter_keyword": [(filter_keyword, filter_keyword)]})
 
-        # get form error messages
-        if error_dict := json.loads(form.errors.as_json()):
-            for dict_values in error_dict.values():
-                for item in dict_values:
-                    form_error_messages.append(item["message"])
-
         if opening_start_date := form.cleaned_data.get("opening_start_date"):
-            # if both dates have valid values but invalid when together
-            if (
-                CUSTOM_ERROR_MESSAGES.get("invalid_date_range")
-                not in form_error_messages
-            ):
-                return_value.update(
-                    {
-                        "opening_start_date": [
-                            (
-                                opening_start_date,
-                                opening_start_date.strftime(
-                                    "Record Opening From: %d-%m-%Y"
-                                ),
-                            )
-                        ]
-                    }
+            return_value["opening_start_date"] = [
+                (
+                    opening_start_date,
+                    "Record opening from: " + opening_start_date.strftime("%d %m %Y"),
                 )
+            ]
 
         if opening_end_date := form.cleaned_data.get("opening_end_date"):
-            # if both dates have valid values but invalid when together
-            if (
-                CUSTOM_ERROR_MESSAGES.get("invalid_date_range")
-                not in form_error_messages
-            ):
```

Comment:
Because 'opening_end_date' will only be present in `form.cleaned_data` if that value is valid. There's no need to check for range validity explicitly here.